docs/website/docs/plus/getting-started/transformations.md

```diff
@@ -0,0 +1,446 @@
+---
+title: Transformations tutorial
+description: Using the dlt+ transformations to transform data
+keywords: [transformations, dlt+, project]
+---
+
+### Topics covered in this tutorial
+
+This tutorial introduces you to the dlt+ transformations basics. You will learn how to:
+
+* transform existing tables with ibis expressions
+* sync datasets, remote to local, local to remote or remote to remote
+* use a local cache to run transformations across multiple datasets
+* retain full column level lineage during syncs and transformations
+
+### Topics covered in a future tutorial
+
+* Incremental transformations
+* Incremental and partial syncing
+* dbt based transformations
+* transformations dlt+ project and CLI integration
+
+## Prerequisites
+
+To follow this tutorial, make sure that:
+
+- dlt+ is set up according to the [installation guide](installation.md)
+- ibis-framework is installed in your environment
+- you're familiar with the [core concepts of dlt](../../reference/explainers/how-dlt-works.md)
+- you're familiar with our read access based on [ibis expressions](../../general-usage/dataset-access/dataset#modifying-queries-with-ibis-expressions)
```

Comment:
question: why 'read' access? that confuses me here....is that something other than using  ibis expressions to access the dataset? is it a different way to say this `methods like limit and select won't work. Include any filtering or column selection directly in your SQL query.`
after the reading the link, I still felt like I didnt check that box